server/controllers/admin.py

```diff
@@ -637,9 +637,19 @@ def enrollment(cid):
     return render_template('staff/course/enrollment.html',
                            enrollments=students, staff=staff,
                            single_form=single_form,
+                           unenroll_form=forms.CSRFForm(),
                            courses=courses,
                            current_course=current_course)
 
+@admin.route("/course/<int:cid>/<string:user_id>/unenroll", methods=['POST'])
+@is_staff(course_arg='cid')
+def unenrollment(cid, user_id):
+    user = User.query.filter_by(id=user_id).one_or_none()
+    if user and user.is_enrolled(cid):
```

Comment:
`user.is_enrolled` actually returns the enrollment object if they are enrolled (though it's hard to tell from the name). 

Given that it returns the enrollment object - we don't need to query for it again in Enroll.unenroll. Instead maybe we can make unenroll a method for enrollment objects. That way we can do something like 
```
enrollment = user.is_enrolled(cid)
if enrollment:
   enrollment.unenroll() #
else:
   ... 
```

There's something else to consider in that all enroll does is a delete - so we don't really need a method for it. We could just call `enrollment.delete()` and be done with it. For now, I'm okay with still making an `unenroll` method since we may want to do other things in the future